apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/testing/testingStore.js

```diff
@@ -1,6 +1,25 @@
 import {create} from "zustand"
 import {devtools} from "zustand/middleware"
 
+const ERRORS_OBJECT= {
```

Comment:
This object is already present in backend. Either get it from there or send the message along with enum.